ui/src/alerting/components/builder/CheckMatchingRulesCard.tsx

```diff
@@ -1,11 +1,70 @@
 // Libraries
-import React, {FunctionComponent} from 'react'
+import React, {FunctionComponent, useState, useEffect} from 'react'
+import {connect} from 'react-redux'
+import {get} from 'lodash'
 
 // Components
-import {EmptyState, ComponentSize} from '@influxdata/clockface'
+import MatchingRuleCard from 'src/alerting/components/builder/MatchingRuleCard'
+import {SpinnerContainer} from '@influxdata/clockface'
+import {TechnoSpinner} from '@influxdata/clockface'
 
-const CheckMatchingRulesCard: FunctionComponent = () => {
-  return (
+// Actions
+import {getActiveTimeMachine} from 'src/timeMachine/selectors'
+
+// API
+import * as api from 'src/client'
+
+//Types
+import {NotificationRule, Check, AppState, CheckTagSet} from 'src/types'
+import {EmptyState, ComponentSize, RemoteDataState} from '@influxdata/clockface'
+
+interface StateProps {
+  check: Partial<Check>
+  orgID: string
+}
+
+const CheckMatchingRulesCard: FunctionComponent<StateProps> = ({
+  check,
+  orgID,
+}) => {
+  const getMatchingRules = async (): Promise<NotificationRule[]> => {
+    const tags: CheckTagSet[] = get(check, 'tags', [])
+    const tagsList = tags.reduce((acc, t) => {
```

Comment:
(real minor) might be easier to read without the `reduce`

```
    const tags: CheckTagSet[] = get(check, 'tags', []).map(t => ['tag', `${t.key}:${t.value}`])
```

Reply:
dammit I failed the chnn map test!

Reply:
fired